.github/workflows/node.js.yml

```diff
@@ -0,0 +1,61 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Since there's no real harm in enabling for all PRs, can we do that? There are some edge cases whether a PR against non-main branch is helpful. Thanks!

```suggestion
```

Ps, sorry, I know you're not asking for review yet, but worried I'll miss the review window since I'm not yet checking notifications every day :)

Reply:
no worries. pls suggest as you them along . Thats the purpose of having draft PR -  to get some early feedback.